src/main/java/Ui.java

```diff
@@ -0,0 +1,167 @@
+package main.java;
+
+import main.java.entity.Task;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * User Interface Management class that helps facilitate
+ * interaction with user
+ * particularly for user input passing
+ * and display query result to user
+ */
+public class Ui {
+    private Scanner sc;
+    static final String GREETINGS = "Dear user, welcome to the world of duke!";
+    static final String PREFIX = "    ";
+    static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    static final String DISPLAY_EMPTY_LIST = PREFIX + "Currently there is nothing on your list.";
+    static final String DISPLAY_ADDED = PREFIX + "Got it. I've added this task:";
+    static final String DISPLAY_DONE = PREFIX + "Nice! I've mark this task as done:";
+    static final String DISPLAY_RENAME = PREFIX + "Noted, I've removed this task:";
+
+    /**
+     * Creates a Ui instance
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Show greetings display
+     */
+    public void greeting() {
+        System.out.println("This is \n" + LOGO);
+        System.out.println(GREETINGS);
+    }
+
+    /**
+     * Show exit display
+     */
+    public void displayExit() {
+        System.out.println(PREFIX + "Bye, until next time!");
+    }
+
+    /**
+     * Show a horizontal line
+     */
+    public void displayLine() {
+        System.out.println("________________________");
+    }
+
+    /**
+     * Read a line of user input
+     * @return a string of the line
```

Comment:
Please leave an empty line between method summary and the parameters section. I noticed this issue in several other places too.